### DISCUSSION TITLE
Using on the pinephone

### DISCUSSION THREAD

**Discussion by henrythemouse** [Igalia/cog#323] [General]
This is a perfect browser for the pinephone!

I've been using it with CogWebRun and couldn't be more impressed with the speed and rendering. Nothing like having the UI separate from the browser, an idea who's time has come. 

Thanks  for providing this.

**Comment by philn** [comment]
BTW, Cog has a GTK4 platform back-end now. We would be happy to review any PR bringing more features to the backend, such as the bookmarks support that CogWebRun has.
